utils/generate_mdschema.py

```diff
@@ -0,0 +1,77 @@
+"""Generate markdown information about tests from the test schema."""
```

Comment:
I think this script should put something like the following either at the start or the end of the resulting file, as a guide to the reader that one shouldn't change the markdown file.

```Generetade with utils/generate_mdschema.py from schema.yaml on DATE```

Maybe even include a note about not editing the file directly but rather just update the underlying schema and run the script again.

Reply:
Added in f20a63b.